cfg/Instructions.h

```diff
@@ -87,13 +87,14 @@ class Send final : public Instruction {
     VariableUseSite recv;
     core::NameRef fun;
     core::LocOffsets receiverLoc;
+    u2 numPosArgs;
```

Comment:
This will be the same value that was present on the corresponding send from ast/Trees.h